tf2onnx/convert.py

```diff
@@ -94,10 +95,14 @@ def main():
         "converted from {}".format(args.input), args.inputs, args.outputs,
         optimize=not args.continue_on_error)
 
+    onnx_graph = OnnxGraph(model_proto.graph)
+    optimizer = TransposeOptimizer(onnx_graph)
```

Comment:
can you make this optional via command line, don't think this tested well enough

Reply:
okay, let me do that.